.github/workflows/node-test.yml

```diff
@@ -69,7 +69,7 @@ jobs:
       fail-fast: false
       matrix:
         node-version:
-          - 8.x
+          - 10.x
```

Comment:
@bkendall if we want to be testing the latest versions of `firebase-admin` we have to make this change.